tasks/internal/request-and-upload-signature/request-and-upload-signature.yaml

```diff
@@ -272,61 +433,84 @@ spec:
           value: $(params.references)
         - name: sig_key_names
           value: "$(params.sig_key_names)"
-        - name: signature_data_file
+        - name: SIGNATURE_DATA_FILE
           value: "$(params.signature_data_file)"
-        - name: requester
+        - name: REQUESTER
```

Comment:
Just curious, why these params need to be changed to be captial letters? But the other params still be left as small letters.

Reply:
when set to lowercase, shellcheck complained about undefined variable